### PR TITLE
Document notification/slack task

### PR DESCRIPTION
**Pull request by kalantar**
Documentation for https://github.com/iter8-tools/iter8/issues/689

### CODE REVIEW

File: mkdocs/docs/reference/tasks/notification.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+template: main.html
+---
+
+# Notification Tasks
+
+## `notification/slack`
+
+### Overview
+
+The `notification/slack` task posts a slack message about current state the experiment.
```
Comment (sriumcp) [inline mkdocs/docs/reference/tasks/notification.md:11]: of the experiment

File: mkdocs/docs/reference/tasks/notification.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+### Examples
+
+The following task in the start action of an experiment creates a notification that will be posted to the slack channel with id `channel` during the execution of the `start` action.
```
Comment (sriumcp) [inline mkdocs/docs/reference/tasks/notification.md:15]: The following task creates a notification (I realize start action is described in this example, but I feel is better to simply describe the task and leave out the action). This task is embeddable into start, loop or finish actions.

**Review by sriumcp — CHANGES_REQUESTED** [review]
Minor changes requested.

**Comment by kalantar** [comment]
Replaced by https://github.com/iter8-tools/iter8/pull/729

**Pull request closed without merging**